packages/permission_handler/example/lib/main.dart

```diff
@@ -50,18 +50,23 @@ class _PermissionHandlerWidgetState extends State<PermissionHandlerWidget> {
                       permission != Permission.accessNotificationPolicy &&
                       permission != Permission.bluetoothScan &&
                       permission != Permission.bluetoothAdvertise &&
-                      permission != Permission.bluetoothConnect;
+                      permission != Permission.bluetoothConnect &&
+                      permission != Permission.nearbyWifiDevices &&
+                      permission != Permission.videos &&
+                      permission != Permission.audio &&
+                      permission != Permission.scheduleExactAlarm &&
+                      permission != Permission.sensorsAlways;
                 } else {
                   return permission != Permission.unknown &&
                       permission != Permission.mediaLibrary &&
-                      permission != Permission.photos &&
```

Comment:
Could you please check this?
In README, photos is iOS-only permission. is this available in tizen?
Please check the permissions allowed by tizen platform, README, and this if condition together.(android-only also..)

Reply:
Permission.photos is for android (with version >= TIRAMISU) and IOS. [permission handler code](https://github.com/Baseflow/flutter-permission-handler/blob/dfed5d2114e4fe1ce2d51711e0a2074b612cb943/permission_handler_platform_interface/lib/src/permissions.dart#L86C1-L101C41)

But it seems there's no privilege related with `photos` in Tizen.
I'll remove `photos` in ios-only list in README, undo removal of that line, and check for other values again.

Reply:
This Tizen plugin returns `kGranted` when given Permission is not convertible into tizen privilege in mobile emulator. 

[Code that converts permission into string](https://github.com/flutter-tizen/plugins/blob/b33b193d5439246ff19437436c6b89b495fae34c/packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc#L21C1-L49C2)
[Code that ignores empty string when requestPermission](https://github.com/flutter-tizen/plugins/blob/b33b193d5439246ff19437436c6b89b495fae34c/packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc#L116C1-L122C12)

So, I think, to test permission supported by test in example app, just not to put the supported permissions into that return statement is enough just like before.

But I'll keep checking there's permissions that convertible into tizen privilege.

Reply:
Fixed it!

Reply:
+ They are ordered by enum value.